docs/content/en/docs/reference/clusterspec/optional/registrymirror.md

```diff
@@ -71,18 +71,19 @@ export REGISTRY_PASSWORD=<password>
 ```
 
 ## Import images into a private registry
-You can use the `import-images` command to pull images from `public.ecr.aws` and push them to your
+You can use the `download images` and `import images` command to pull images from `public.ecr.aws` and push them to your
 private registry.
-Starting with release 0.8, import-images command also pulls the cilium chart from `public.ecr.aws` and pushes it to the registry mirror. It requires the registry credentials for performing a login. Set the following environment variables for the login:
+The `download images` command also pulls the cilium chart from `public.ecr.aws` and pushes it to the registry mirror. It requires the registry credentials for performing a login. Set the following environment variables for the login:
 ```bash
 export REGISTRY_USERNAME=<username>
 export REGISTRY_PASSWORD=<password>
 ```
 
 ```bash
+eksctl anywhere download images -o images.tar
```

Comment:
I was wondering if `images.tar` is the literal name representing a tarball of all EKS Anywhere images or if that's just meant to represent a particular image name (like harbor.tar). I think it would be good to mention exactly what it contains or if it needs to be replaced by particular names.

Reply:
```suggestion
eksctl anywhere download images -o eks-anywhere-images.tar
```